asapdiscovery-data/asapdiscovery/data/openeye.py

```diff
@@ -5,6 +5,78 @@
     raise RuntimeError("OpenEye license required to use asapdiscovery openeye module")
 
 
+def combine_protein_ligand(prot, lig, lig_name="LIG", resid=None, start_atom_id=None):
+    """
+    Combine a protein OEMol and ligand OEMol into one, handling residue/atom
+    numbering, and HetAtom status.
+
+    Parameters
+    ----------
+    prot : oechem.OEMol
+        OEMol with the protein atoms. This should have perceived resiudes
+    lig : oechem.OEMol
+        OEMol with the ligand atoms
+    lig_name : str, default="LIG"
+        Residue name to give to the ligand atoms
+    resid : int, optional
+        Which residue number to assign to the ligand. If not given, the largest existing
+        residue number in `prot` will be found, and the ligand will be assigned the next
+        number
+    start_atom_id : int, optional
+        Which atom number to assign to the first atom in the ligand. If not given, the
+        next available atom number will be calculated and assigned
+
+    Returns
+    -------
+    oechem.OEMol
+        Combined molecule, with the appropriate biopolymer field set for the lig atoms
+    """
+    # Calculate residue number if necessary
+    if resid is None:
+        # Find max resid for numbering the ligand residue
+        resid = max([r.GetResidueNumber() for r in oechem.OEGetResidues(prot)]) + 1
```

Comment:
would probably be good to document why all these +1s are necessary in the docstring. I assume it is a 0 vs 1 based indexing thing, but good for someone following later.

Reply:
added in [c1cf66f](https://github.com/choderalab/asapdiscovery/pull/160/commits/c1cf66f8eec4eda12987b36a387ff10bbbb02dfd)